modules/multi-architecture-modify-machine-set-aws.adoc

```diff
@@ -97,11 +97,11 @@ spec:
 +
 [source,terminal]
 ----
-$ oc get -o jsonpath=‘{.status.infrastructureName}{“\n”}’ infrastructure cluster
+$ oc get -o jsonpath="{.status.infrastructureName}{'\n'}" infrastructure cluster
 ----
 <2> Specify the infrastructure ID, role node label, and zone.
 <3> Specify the role node label to add.
-<4> Specify a Red{nbsp}Hat Enterprise Linux CoreOS (RHCOS) Amazon Machine Image (AMI) for your AWS zone for the nodes. The RHCOS AMI must be compatible with the machine architecture.
+<4> Specify a Red{nbsp}Hat Enterprise Linux CoreOS (RHCOS) Amazon Machine Image (AMI) for your AWS region for the nodes. The RHCOS AMI must be compatible with the machine architecture.
```

Comment:
```suggestion
<4> Specifies a Red{nbsp}Hat Enterprise Linux CoreOS (RHCOS) Amazon Machine Image (AMI) for your AWS region for the nodes. The RHCOS AMI must be compatible with the machine architecture.
```

Reply:
Same comment.

Using "Specifies" is not required here. We are asking users to specify their value for a given placeholder.